tests/e2e/Services/Users/UsersBase.php

```diff
@@ -926,6 +926,62 @@ public function testUpdateAndGetUserPrefs(array $data): array
         return $data;
     }
 
+    /**
+     * @depends testGetUser
+     */
+    public function testUpdateUserNumber(array $data): array
+    {
+        /**
+         * Test for SUCCESS
+         */
+        $updatedNumber = "+910000000000"; //dummy number
+        $user = $this->client->call(Client::METHOD_PATCH, '/users/' . $data['userId'] . '/phone', array_merge([
+            'content-type' => 'application/json',
+            'x-appwrite-project' => $this->getProject()['$id'],
+        ], $this->getHeaders()), [
+            'number' => $updatedNumber,
+        ]);
+
+        $this->assertEquals($user['headers']['status-code'], 200);
+        $this->assertEquals($user['body']['phone'], $updatedNumber);
+
+        $user = $this->client->call(Client::METHOD_GET, '/users/' . $data['userId'], array_merge([
+            'content-type' => 'application/json',
+            'x-appwrite-project' => $this->getProject()['$id'],
+        ], $this->getHeaders()));
+
+        $this->assertEquals($user['headers']['status-code'], 200);
+        $this->assertEquals($user['body']['phone'], $updatedNumber);
+
+        return $data;
```

Comment:
Would you please add a:

```
        /**
         * Test for FAILURE
         */
```

block that updates the phone number to an existing phone number and verifies the error response returned? This is to test the code change you made.

Reply:
Sure

Reply:
@stnguyen90 Done, please check.